governance/votes/Executive vote - November 28, 2024.md

```diff
@@ -0,0 +1,267 @@
+
+- Transfer **3 million USDS** to [0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7](https://etherscan.io/address/0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7).
+
+### ESM Minimum Threshold Increase
+
+- **Authorization**: [Governance Facilitator Authorization](https://forum.sky.money/t/november-28th-spell-parameter-proposal-esm-threshold-gsm-delay-increase/25579/2)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/november-28th-spell-parameter-proposal-esm-threshold-gsm-delay-increase/25579)
+
+If this executive proposal passes, then the [Emergency Shutdown Module (ESM)](https://docs.makerdao.com/smart-contract-modules/shutdown/emergency-shutdown-module) [Minimum Threshold (`min`)](https://manual.makerdao.com/module-index/module-emergency-shutdown#minimum-threshold-min) will be increased by 200,000 MKR from 300,000 MKR to **500,000 MKR**.
+
+
+### Aligned Delegate Compensation for October 2024
+
+- **Authorization**: [Atlas A.1.5.8](https://sky-atlas.powerhouse.io/A.1.5.8_Budget_For_Prime_Delegate_Slots/e3e420fc-9b1f-4fdc-9983-fcebc45dd3aa%7C0db3af4ece0c)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/october-2024-aligned-delegate-compensation/25581)
+
+If this executive proposal passes, then a total of **20,701 USDS** will be distributed to six Aligned Delegates and their teams as compensation for October 2024.
+
+| Delegate | Amount (USDS) | Address |
+|---|---|---|
+| BONAPUBLICA | 4,000 | [0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3) |
+| Cloaky | 4,000 | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) |
+| JuliaChang | 4,000 | [0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7](https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7) |
+| Vigilant | 4,000 | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) |
+| BLUE | 2,968 | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
+| Byteron | 1,733 | [0xc2982e72D060cab2387Dba96b846acb8c96EfF66](https://etherscan.io/address/0xc2982e72D060cab2387Dba96b846acb8c96EfF66) |
+
+### Atlas Core Development Payment to Key Atlas Contributors
+
+- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580/8), Atlas Axis Authorizations [1](https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580/4), [2](https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580/6)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/atlas-core-development-payment-requests-november-2024/25580)
+
+If this executive proposal passes, then a total of **90,277 USDS** and **330,000 SKY** will be distributed to three Atlas Core Development key contributors.
+
+| Contributor | Amount | Address |
```

Comment:
For verification, the ordering here should match the order on the Exec sheet. Non-blocking